scripts/check_toc_is_complete.py

```diff
@@ -5,16 +5,16 @@
 You can run it manually with `pre-commit run check-toc --all`.
 """
 
-import json
 from pathlib import Path
 import argparse
+import ast
 
 if __name__ == "__main__":
     toc_examples = (Path("docs") / "source/notebooks/table_of_contents_examples.js").read_text()
     toc_tutorials = (Path("docs") / "source/notebooks/table_of_contents_tutorials.js").read_text()
     toc_keys = {
-        **json.loads(toc_examples[toc_examples.find("{") :]),
```

Comment:
what was wrong with th eold way?

Reply:
if you included a trailing comma then you'd get a mysterious
```
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 69 column 1 (char 3689)
```
, while `ast.literal_eval` can handle trailing commas